.travis.yml

```diff
@@ -36,21 +36,13 @@ install:
         make --no-print-directory -s pkg-utils pkg-kmod;
         sudo dpkg -i *.deb;
       fi
-    - cd ../ZoL
+    - cd ../zfs
     - sh autogen.sh
     - if [ $ZFS_BUILD_TAGS = 0 ]; then
-        if [ $TRAVIS_BRANCH = "zfs-0.7-release" ]; then
-          ./configure --enable-code-coverage=yes --enable-uzfs=yes --with-jemalloc || travis_terminate 1;
-        else
-          ./configure --enable-code-coverage=yes --enable-debug --enable-uzfs=yes --with-jemalloc || travis_terminate 1;
-        fi;
+        ./configure --enable-code-coverage=yes --enable-uzfs=yes --with-jemalloc || travis_terminate 1;
         make;
       else
-        if [ $TRAVIS_BRANCH = "zfs-0.7-release" ]; then
-          ./configure --enable-code-coverage=yes || travis_terminate 1;
-        else
-          ./configure --enable-code-coverage=yes --enable-debug || travis_terminate 1;
-        fi;
+        ./configure --enable-code-coverage=yes || travis_terminate 1;
```

Comment:
I would add --enable-debug to both configure commands. We want to uncover as many bugs as possible on behalf of these tests and performance measurements are not of interest to us.

Reply:
@jkryl .. right.. now, being cstor repo is the one who creates the images, we can make it enable-debug in this repo's travis file. Will do the change